packages/graphql-yoga/src/index.ts

```diff
@@ -99,27 +76,29 @@ export class GraphQLServer extends BaseNodeGraphQLServer {
     const envelop = this.envelop
     this.logger.debug('Setting up server.')
 
+    this._server.addContentTypeParser('multipart/form-data', (request, body, done) => {
+      done(null)
+    })
+
     this._server.route({
       method: ['GET', 'POST'],
       url: this.endpoint,
-      async handler(req, res) {
+      async handler(req, reply) {
         const request = await getNodeRequest(req)
 
         if (shouldRenderGraphiQL(request)) {
-          res.raw.end(renderGraphiQL())
+          reply.type("text/html");
+          reply.send(renderGraphiQL({}));
         } else {
-          const result = await handler(request, schema, envelop)
-
-          /**
-           * Get headers from Fastify response to raw response
-           * Workaround for helix to use fastify middleware
-           * @see https://github.com/contra/graphql-helix/issues/75#issuecomment-976499958
-           */
-          for (const [key, value] of Object.entries(res.getHeaders())) {
-            res.raw.setHeader(key, String(value || ''))
-          }
-
-          return sendNodeResponse(result, res.raw)
+          const response = await handler(request, schema, envelop)
+
+          response.headers.forEach((value, key) => {
+            reply.header(key, value);
+          });
+
+          reply.status(response.status);
+          reply.send(response.body);
+          reply.sent = true;
```

Comment:
sen.true must not be set here as we are not messing around with the raw response object

Reply:
👍